plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -330,7 +331,8 @@ func ClusterRoles() []rbac.ClusterRole {
 				rbac.NewRule("get", "update", "patch", "delete").Groups(legacyGroup).Resources("endpoints").Names("kube-scheduler").RuleOrDie(),
 
 				// fundamental resources
-				rbac.NewRule(Read...).Groups(legacyGroup).Resources("nodes", "pods").RuleOrDie(),
+				rbac.NewRule(Read...).Groups(legacyGroup).Resources("nodes").RuleOrDie(),
+				rbac.NewRule(ReadDelete...).Groups(legacyGroup).Resources("pods").RuleOrDie(),
```

Comment:
I'd probably just enumerate `"get","list","watch","delete"` here... ReadDelete is an unusual combination so I don't expect it would be commonly used